interop/xds/server/server.go

```diff
@@ -36,6 +36,8 @@ var (
 	port     = flag.Int("port", 8080, "The server port")
 	serverID = flag.String("server_id", "go_server", "Server ID included in response")
 	hostname = getHostname()
+
+	logger = grpclog.Component("interop")
```

Comment:
Would it make sense to have a `logging.go` file in `intertop/` and define the logger there?

Reply:
Creating this component only takes a reference to the existing component (if it always exists), so the overhead is small.

The `interop` package has too many things that aren't needed by all tests.